data/data2.s

```diff
@@ -22977,7 +23254,7 @@ gUnknown_083FA608:: @ 83FA608
 	.incbin "baserom.gba", 0x003fa608, 0x6
 
 gUnknown_083FA60E:: @ 83FA60E
-	.incbin "baserom.gba", 0x003fa60e, 0x8
+	.2byte 0x74, 0x2C, 0x74, 0x70
```

Comment:
these seem to be coordinates. so there should be 2 per line and I all other coordinates are in decimal rather than hex